rust/streaming_pipe_in_and_out.rs

```diff
@@ -0,0 +1,20 @@
+// Script reads text from a named pipe and writes it another named pipe, capitalized
+use std::env;
+use std::fs::File;
+use std::io::{self, BufRead, BufWriter, Write};
```

Comment:
Let's remove 
```suggestion
use std::io::{BufReader, BufRead, BufWriter, Write};
```
And below, we would need to do `BufReader::new` insead of `io::BufReader::new`